### PR TITLE
Fixes for dt usage in update()

### PR DESCRIPTION
**Pull request by julesghub**
@rbeucher I think these need to be changed, time stepping models were failing in the update(). 
This changes fixes things I believe.

### CODE REVIEW

File: UWGeodynamics/_model.py [hunk 1]
```diff
@@ -1781,40 +1782,40 @@ def _update(self):
...
         self.population_control.repopulate()
         self.swarm.update_particle_owners()
 
         if self.surfaceProcesses:
-            self.surfaceProcesses.solve(dt.value[0])
+            self.surfaceProcesses.solve(dt)
 
         # Update Time Field
-        self.timeField.data[...] += dt.value[0]
+        self.timeField.data[...] += dt
```
Comment (rbeucher) [inline UWGeodynamics/_model.py:1815]: Not sure about that one...

**Comment by julesghub** [comment]
Awesome. I'll delete the hotfix branch

**Comment by rbeucher** [comment]
I have updated the tag... The M1 docker image needs updating to

**Comment by julesghub** [comment]
ok, is that a manual operation still?

**Comment by rbeucher** [comment]
Yes it is...sighs...

**Pull request merged**